### ISSUE TITLE
Translation problem

### ISSUE THREAD

**Issue by ArchdukeNavaron** [CachetHQ/Cachet#3246]
Hi,
I am experiencing some problems whit the german translation.
The button "next" or "previous" is shown as follows:
![2018-09-17 16_39_28-window](https://user-images.githubusercontent.com/33832625/45630111-4928b500-ba98-11e8-9424-22757cc2d0f5.png)

Do you see the problem?
It adds a &laquo; and a &raquo;.
Who am I ablo to remove them? 

They don't show up in the original english version.

**Comment by jbrooksuk** [comment]
@ArchdukeNavaron what version of Cachet are you using? In the 2.4 files, there isn't any HTML in the string https://translate.cachethq.io/proofread/cachet/25/en-de

**Comment by jbrooksuk** [comment]
Can you run `git pull`

**Comment by ArchdukeNavaron** [comment]
@jbrooksuk 
After I executed git pull, I had some issues with the composer.lock file.
After resolving them and finally executing it again, i got some problems with dependencies.
php artisan app:update did not work as packages like barryvdh/laravel-debugbar or Roumen\Feed\FeedServiceProvider were missing. As i didn't want to add them one by one I decided to roll back.

Is this a known problem or is there anything else going or am I doing anything wrong?
